packages/cli/api-importers/v2-importer-commons/src/converters/abstract/AbstractParameterConverter.ts

```diff
@@ -176,4 +183,35 @@ export abstract class AbstractParameterConverter<
                 : userSpecifiedExamples;
         return userSpecifiedExamples;
     }
+
+    private async generateOrValidateExample({
+        schema,
+        shouldCollectErrors,
```

Comment:
Thoughts on making that `optional` `ignoreErrors` parameter? I think that will clean up a lot of the call-sites.